pytorch_lightning/distributed/dist.py

```diff
@@ -40,6 +41,6 @@ def broadcast(self, obj: Any, group=_group.WORLD):
         if self.rank != 0:
             obj = [None] * len(obj)
 
-        broadcast_object_list(obj, 0, group=group or _group.WORLD)
+        torch.distributed.broadcast_object_list(obj, 0, group=group or _group.WORLD)
```

Comment:
Let's keep `broadcast_object_list` for now as Habana Plugin is modifying it.

Reply:
Habana should not modify the implementation as it's supposed to be a clone of what's in torch.

They should instead provide their own implementation which could be easily used after the collectives refactor.

Reply:
Yup, I mentioned it to override it in Habana's own DDP Plugin rather than modifying this